src/main/java/com/microsoft/aad/msal4j/HttpUtils.java

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+import java.util.List;
+import java.util.Map;
+
+class HttpUtils {
+
+    static String headerValue(Map<String, List<String>> headers, String headerName){
+
+        if(headerName == null || headers == null){
+            return null;
+        }
+
+        List<String> headerValue = headers.get(headerName);
+
+        if(headerValue == null || headerValue.isEmpty()){
+            return null;
+        }
+
+        return String.join(",", headerValue);
```

Comment:
>String.join(",", headerValue); [](start = 15, length = 30)

couldn't there be other separators?

Can you pls. add tests for this?

Reply:
Not sure I understand your question. To clarify,  `HttpUtils` class is only used when we need to log certain headers returned from HTTP responses for logs or telemetry.  Since headers are returned as a list, what we are doing here is in certain cases where the list has multiple values (only case that I've seen from testing is the "Set-Cookie" header, which we don't even log anywhere), then we join those multiple values into one string, delimited by a comma. We could use any delimiter we want, but comma seemed like a sensible choice.